books/RayTracingTheRestOfYourLife.html

```diff
@@ -353,59 +466,93 @@
-enough. Light sources are only sampled if a ray scatters toward them, but this can be unlikely for a
-small light, or a light that is far away. We could lessen this problem if we sent more random
-samples toward this light, but this will cause the scene to be inaccurately bright. We can remove
-this inaccuracy by downweighting these samples to adjust for the over-sampling. How we do that
-adjustment? To do that, we will need the concept of a _probability density function_.
+This, as expected, produces approximately the exact answer we get with integration, i.e. $I = 8/3$.
+You could rightly point to this example and say that integration is actually a lot less work. That
+might be true in the case where the function is $f = x^2$, but there exists lots of functions where
+it might be simpler to solve for the MC than for the integration, like $f = sin^5(x)$. We could also
+use the MC for functions where an analytically integration does not exist, like $\log(\sin(x))$. In
+graphics, we often have functions that we can write down explicitly but that have a complicated
+analytic integration, or, just as often, we have functions that can be evaluated but that can't be
+written down explicitly, and we will frequently find ourselves with a function that can _only_ be
+evaluated probabilistically. The function `ray_color` from the first two books is an example of a
+function that can only be determined probabilistically. We can’t know what color can be seen from
+any given place in all directions, but we can statistically estimate which color can be seen from
+_this_ place and in _a_ direction.
 
 
 Density Functions
 ------------------
 <div class='together'>
-First, what is a _density function_? It’s just a continuous form of a histogram. Here’s an example
+The `ray_color` function that we wrote in the first two books, while elegant in it's simplicity, has
+a fairly _major_ problem. Small light sources create too much noise. This is because our uniform
+sampling doesn’t sample these light sources often enough. Light sources are only sampled if a ray
+scatters toward them, but this can be unlikely for a small light or a light that is far away. If
+the background color is black, then the only real sources of light in the scene are from the lights
+that are actually placed about the scene. So there might be two rays that intersect next to each
+other on a material, one that is randomly reflected toward the light and one that is not. The ray
+that is reflected toward the light will appear a very bright color, whereas the other ray will
+appear very dark. When the two of them should really be somewhere in the middle. We could lessen
+this problem if we sent _more_ random samples toward this light, but this will cause the scene to be
+inaccurately bright.
+
+We usually trace rays from the camera out into the scene, but imagine if we traced rays from the
+light source out into the source. It would start as a very bright ray, which would lose energy with
```

Comment:
“... out into the scene. They would start with their original intensities, which would ... colored by their reflections ...”

Alternately, make everything singular: “but imagine if we traced a ray from ...”

Generally in this chapter make the example consistently singular or consistently plural.